itou/siaes/management/commands/_import_siae/convention.py

```diff
@@ -65,6 +67,12 @@ def update_existing_conventions():
             convention.save()
 
         should_be_active = does_siae_have_an_active_convention(siae)
+
+        # TODO @dejafait DROP after 2023/05/31
+        # 199 SIAE were deactivated on 2023/04/24 but per ASP's orders we have to keep them artifically alive
+        # until at least 2023/05/31.
+        should_be_active = should_be_active or convention.deactivated_at.date() == datetime.date(2023, 4, 24)
```

Comment:
Peut-être rajouté un
```suggestion
        should_be_active = should_be_active or (convention.deactivated_at.date() == datetime.date(2023, 4, 24) and timezone.localdate() < datetime.date(2023, 6, 1))
```

Reply:
Pas d'accord. Là ils nous disent le 31 mai mais à tous les coups ils vont encore retarder. J'ai pas envie d'introduire une bombe à retardement dans le code. @xavfernandez

Reply:
C'est ce que je me suis dit aussi, je pense que c'est trop flou et je préfère laisser des SIAE capables d'opérer plutot que les bloquer.

Par contre ce que je me demande plutot c'est : est-ce qu'on ne voudrait pas plutôt:
- retirer leur `deactivated_at` complètement (quitte à enregistrer leurs PKs quelque part)
- ne plus faire de `_import_siae` jusque en Juin ? Parce que franchement l'ASP là ils abusent. Leur retard etait jusque en Fevrier, puis Mars, puis Avril, maintenant ils nous disent carrément de reconventionner tout le monde et d'ignorer ce qu'ils nous envoient.
Donc ?

Reply:
Sinon sur le papeir je suis d'accord avec le fix, mais en vrai pour moi ça pose la question de cette synchro. Si demain ca va jusqu'en Septembre d'ignorer les retours de déconventionnement ASP, à quoi bon ?